app/Libraries/OsuAuthorize.php

```diff
@@ -891,6 +891,13 @@ public function checkUserFavouriteRemove($user)
         return 'ok';
     }
 
+    public function checkUserReport($user)
+    {
+        $this->ensureCleanRecord($user);
```

Comment:
why is a clean record required for this?

Reply:
don't think this is required